src/fparser/two/Fortran2008.py

```diff
@@ -526,6 +526,20 @@ class Upper_Cobound(Base):  # R513
     subclass_names = ['Specification_Expr']
 
 
+class Error_Stop_Stmt(StmtBase, WORDClsBase):  # R856
+    '''
+    Fortran 2008 rule R856
+    error-stop-stmt is ERROR STOP [ stop-code ]
+
+    '''
+    subclass_names = []
+    use_names = ['Stop_Code']
+
+    def match(string):
```

Comment:
Please could you change this to use the `@staticmethod` decorator instead of the code on line 540.

Reply:
Please could you also add a docstring for this method and in it document the argument and return values and types (see the `match` method below at line 565 for an example).